test/rating.js

```diff
@@ -16,40 +16,13 @@ describe('A track in Google Music', function () {
   });
   browserUtils.execute(function playViaApi () {
     window.googleMusic.playback.playPause();
+    window.googleMusic.playback.toggleRepeat();
```

Comment:
Yea, at some point we need to standardize the API to be consistent though =/ (e.g. only provide `set`/`get`, always provide `set`/`get`/`toggle` (where possible))